File: app/src/main/java/com/estrrado/tradopay/ui/app/TradoPay.java
package com.estrrado.tradopay.ui.app;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

public class TradoPay extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(TradoPay.this);
    }
}

File: app/src/main/java/com/estrrado/tradopay/app/TradoPay.kt
package com.estrrado.tradopay.ui.app

import android.app.Application
import com.facebook.drawee.backends.pipeline.Fresco

class TradoPay : Application() {
    override fun onCreate() {
        super.onCreate()
        Fresco.initialize(this@TradoPay)
    }
}